v2/cards/anvil.json

```diff
@@ -0,0 +1,16 @@
+{
+  "card_type_id": "ice",
+  "cost": 4,
+  "deck_limit": 3,
+  "faction_id": "weyland_consortium",
+  "id": "anvil",
+  "influence_cost": 2,
+  "is_unique": false,
+  "side_id": "corp",
+  "strength": 3,
+  "stripped_text": "When the Runner encounters this ice, you may trash 1 of your other installed cards. If you do, the Runner cannot break this ices printed subroutines for the remainder of this encounter. Subroutine Gain 1 credit. The Runner loses 1 credit. Subroutine The Runner trashes 1 of their installed cards.",
```

Comment:
"ices" should be "ice's"
I don't think we remove apostrophes in stripped text (at least, I've found counter examples in the existing data)

Reply:
hmm.  Yeah, we probably have more instances of that.  I'll file an issue to track it.